tools/fmt.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd "${0%/*}/.."
+ROOT="$(pwd)"
+
+shfmt -l -i 2 -sr -kp -w ${*:-packages/*/PKGBUILD}
```

Comment:
When and by whom (manual/CI) should this be used? Maybe add a short explanation or quickstart section to the readme, including any depenedencies, which need to be installed locally (if any)?

Reply:
ee8828510fe322f3432229444a3a3a64d4c69a1d